scripts/update_docs_version_selector.py

```
@@ -0,0 +1,262 @@
+    parts: List[int] = []
+    for piece in version.split("."):
+        try:
+            parts.append(int(piece))
+        except ValueError:
+            parts.append(0)
+    return tuple(parts)
+
+
+def fetch_gh_pages(repo_root: Path) -> None:
+    result = subprocess.run(
+        ["git", "fetch", "origin", "gh-pages"],
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)